src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -50,9 +50,24 @@ let private increaseContribution valueIncrease aidValue (aidMap: Dictionary<AidH
 
   aidMap.[aidHash] <- updatedContribution
 
-type private RequestedOutput =
-  | Value
-  | Noise
+let private mergeOutliers outliers (contributions: Anonymizer.ContributionsState array) =
+  outliers
+  |> Array.iteri (fun aidIndex aidOutliers ->
+    let aidContributions = contributions.[aidIndex].AidContributions
+
+    aidOutliers
+    |> Array.iter (fun (aid, contribution) ->
```

Comment:
in light of the other comment from `Anonymizer.fs`, this could be `(aid, flattening)` - if I am following the variables correctly

Reply:
This is the left-over contribution, I think "droppedContribution" makes more sense.